tests/unit/amo/components/TestSearchResult.js

```diff
@@ -146,17 +146,56 @@ describe(__filename, () => {
     expect(root).toHaveClassName('SearchResult--theme');
   });
 
-  it('displays a message if the theme preview image is unavailable', () => {
+  it('displays a message if the lightweight theme preview image is unavailable', () => {
     const addon = createInternalAddon({
       ...fakeAddon,
+      // The 'previews' field is not currently being used by lightweight themes
+      // So here we are just overridding the fakeAddon values to mimic the API
+      // response.
+      previews: [],
+      theme_data: {
+        previewURL: null,
+      },
+      type: ADDON_TYPE_THEME,
+    });
+    const root = render({ addon });
+
+    expect(root.find('.SearchResult-notheme'))
+      .toIncludeText('No theme preview available');
+  });
+
+  it("does not display a 'no theme preview available' message if the lightweight theme preview image is available", () => {
+    const addon = createInternalAddon({
+      ...fakeTheme,
       type: ADDON_TYPE_THEME,
     });
     const root = render({ addon });
 
+    expect(root.find('.SearchResult-icon')).toHaveLength(1);
```

Comment:
maybe we should assert for `not.toIncludeText()` since that's what the test case description says?